### DISCUSSION TITLE
open link in side panel

### DISCUSSION THREAD

**Discussion by DevrayePatil** [Budibase/budibase#7144] [Help]
I want to make the ui that can be open in side panel, how can I get this
When I click on any link that should be open in side panel

**Comment by melohagan** [comment]
Hey @DevrayePatil 

Would you be able to provide a little more context please? Some screenshots would help me out too!

**Reply by DevrayePatil** [reply]
![image](https://user-images.githubusercontent.com/54846160/183567299-3aa70bf1-6d0f-498c-8454-43440308c452.png)
![image](https://user-images.githubusercontent.com/54846160/183567375-843dd819-111f-46c2-9624-66db5a403284.png)

when I click on any card that information should be open in the side panel as shown in the above screenshots
How can I get this

**Reply by melohagan** [reply]
Brilliant, thanks @DevrayePatil 

Have you had a look at conditional UI? https://docs.budibase.com/docs/conditions

You can use <a href="https://docs.budibase.com/docs/app-state">App State</a> to keep track of which panel is  open, e.g. `openPanel = "switch-off-energy-isolator"`

Kudos to **poirazis** for providing an example! https://github.com/Budibase/budibase/discussions/5763#discussioncomment-2843002
